tests/ui/coherence/coherence-conflicting-repeated-negative-trait-impl-70849.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(negative_impls)]
+//@ edition: 2021
+// Test to ensure we are printing the polarity of the trait when printing out
+// conflicting trait impls
```

Comment:
```suggestion
// Test to ensure we are printing the polarity of the impl trait ref
// when printing out conflicting trait impls
```

(minor) A trait can't have a polarity but an *impl trait ref(erence)* can (basically, trait ref = trait name + generic args like `Trait<'a>`)

Reply:
Yup that makes sense. Thanks for the clarification!